src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.search_form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search_bar {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  background: url(./images/Search.svg) no-repeat scroll 26px 25px;
+  padding-left: 62px;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search_bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search_bar::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search_bar:focus {
+  background: url(./images/Search.svg) 26px 25px no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+}
+
+.small_bar {
+  height: 42px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+}
+
+.search_bar_small:focus {
+  background: url(./images/Search.svg) 13px 15px no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-size: 11px;
+  font-size: 14px;
```

Comment:
don’ t duplicate styles,  set  for focus only the styles that differs from styles in class .small_bar